README.md

```diff
@@ -333,10 +333,35 @@ RUN apk add --no-cache \
 ENV PANTHER_CHROME_DRIVER_BINARY /usr/lib/chromium/chromedriver
 ```
 
+### GitHub Actions Integration
+
+Panther works out of the box with [GitHub Actions](https://help.github.com/en/actions).
+Here is a minimal `.github/workflows/panther.yml` file to run Panther tests:
+
+```yaml
+name: Run Panther tests
+
+on: [ push, pull_request ]
+
+jobs:
+  tests:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: composer install -q --no-ansi --no-interaction --no-scripts --no-suggest --no-progress --prefer-dist
```

Comment:
is `composer` actually preinstalled ? Or do you need the `setup-php` action before that ?

Reply:
It is preinstalled, you don't need anything: https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md